src/main/java/edu/kis/powp/jobs2d/features/DriverFeature.java

```diff
@@ -33,7 +33,7 @@ public static void setupDriverPlugin(Application application) {
      * @param name   Button name.
      * @param driver Job2dDriver object.
      */
-    public static void addDriver(String name, Job2dDriver driver) {
+    public static void addDriver(String name, LineDriverAdapter driver) {
```

Comment:
Adding drivers should not be limited to LineDriverAdapter. If you need to make some additional assumptions about drivers used in DriverManager it should be done in more abstract and OCP way (e.g., introduce new Job2dDriver sub-interface)